json-schema-form.d.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * Shorthand to lookup for keys with `x-jsf-*` preffix.
+ */
+export function pickXKey(node: Object, key: 'presentation' | 'error-message'): Object | undefined;
+
+type JSConfig = {
+  initialValues?: Record<string, unknown>;
+  strictInputType?: boolean;
+  customProperties?: {
+    description: Function | string;
+    [key: string]: unknown;
+  };
+  inputTypes?: {
+    errorMessage: Record<string, unknown>;
+  };
+};
+
+type Fields = Record<string, unknown>[]; // TODO: We don't know what type we have here, we need to investigate
+
+type $TsFixMe = any;
```

Comment:
nit: Could we name it `$TSFixMe` to match our internal codebase?

Reply:
Done!